Lombiq.OrchardCoreApiClient/Models/TenantSetupApiModel.cs

```diff
@@ -14,6 +14,7 @@ public class TenantSetupApiModel
     public string DatabaseProvider { get; set; }
     public string ConnectionString { get; set; }
     public string TablePrefix { get; set; }
+    public string Schema { get; set; }
```

Comment:
Is this actually used somewhere or only here for parity?

Reply:
Only for parity, if anybody needs it.